src/gestalts/GestaltGraph.ts

```diff
@@ -172,11 +174,11 @@ class GestaltGraph {
           const vertexKeys = gKs;
           gestalt.matrix[vertex] = vertexKeys;
           if (gId.type === 'node') {
-            const nodeInfo = await this.db.get<NodeInfo>(
+            const gestaltNodeInfo = await this.db.get<GestaltNodeInfo>(
```

Comment:
What was the reason for changing this type?

Reply:
The Id doesn't convert back properly if it's if it's part of an object. so I had to convert it to a string before putting it into the DB.

Reply:
Discussed here: https://github.com/MatrixAI/js-polykey/pull/318#discussion_r793264040